### DISCUSSION TITLE
Add dark/light mode switch

### DISCUSSION THREAD

**Discussion by mahmoudhossam** [MonitorControl/MonitorControl#1481] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Apple buried the dark mode switch under control center -> display -> dark mode, or you can go through system settings which is quite annoying

### Describe the solution you'd like

I'd like to be able to toggle between dark and light modes fromMonitorControl

### Describe alternatives you've considered

I considered adding the display panel to the menu bar, but that clutters an already busy corner of my screen even more

**Comment by waydabber** [comment]
I added this feature to [BetterDisplay](https://betterdisplay.pro) (you can also define a keyboard shortcut for it), I advise you to try that app. There are no plans to add additional features to MonitorControl as it is considered feature-complete by the team.

**Comment by waydabber** [comment]
<img width="390" alt="Screenshot 2023-12-12 at 15 52 14" src="https://github.com/MonitorControl/MonitorControl/assets/37590873/106fedd5-4190-469c-bb05-8cc3fb65f1b0">
